### ISSUE TITLE
Navbar is completely invisible on mobile

### ISSUE THREAD

**Issue by remram44** [ViDA-NYU/reprozip-examples#31]

**Comment by remram44** [comment]
If I compare to the main website, it's missing something like:
```diff
--- a/.bootstrap/css/custom.css
+++ b/.bootstrap/css/custom.css
@@ -42,6 +42,14 @@ main {
        background-color: blue;
 }
 
+.navbar-inverse .navbar-toggler {
+  border: 1px solid white;
+}
+
+.navbar-inverse .navbar-toggler-icon {
+  background-image: url("data:image/svg+xml,%3csvg viewBox='0 0 30 30' xmlns='http://www.w3.org/2000/svg'%3e%3cpath stroke='rgba(255, 255, 255, 1)' stroke-width='2' stroke-linecap='round' stroke-miterlimit='10' d='M4 7h22M4 15h22M4 23h22'/%3e%3c/svg%3e");
+}
+
 .nav.navbar-nav.ml-auto > .nav-item > .nav-link {
   font-size:2em;
 }
```
Though I'm not sure why this is in `custom.css` in the first place? Isn't it built into bootstrap?

**Comment by remram44** [comment]
Fixed by 63d30e60
